### PR TITLE
Add setAirplaneMode action

### PR DESCRIPTION
**Pull request by pietropizzi**
Let's start with something simple here ✈️ 

Really like this!

### CODE REVIEW

File: src/actions/setAirplaneMode.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+type SetAirplaneModeOptions = {
+  value?: boolean;
+};
+
+/**
+ * Set Airplane Mode Action. Sets the device's Airplane Mode to on or off.
+ * @param {Object} options
+ * @param {boolean} [options.value=true]
+ */
+const setLowPowerMode = (
```
Comment (joshfarrant) [inline src/actions/setAirplaneMode.ts]: Can this be changed to setAirplaneMode please, along with the export?

(I've lost track of the number of times I've done this when adding new actions 🙃!)

**Comment by joshfarrant** [comment]
Fantastic, thanks for the PR! One little tweak then I think it's good to go! ✈️

**Comment by joshfarrant** [comment]
A model PR, btw ⭐️Great way to start things off!

File: src/actions/setAirplaneMode.ts (hunk 1 above)
Comment (pietropizzi) [reply src/actions/setAirplaneMode.ts]: Haha, classic copy pasta. Will update.

**Comment by joshfarrant** [comment]
Looks great, thank you so much for taking the time to add this!

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by joshfarrant** [comment]
@pietropizzi I referenced you in #6, hope that's ok!

**Comment by pietropizzi** [comment]
Sweet!

**Pull request merged**